remote-finetuning.md

```diff
@@ -0,0 +1,128 @@
+1. Display the job streaming log if the job has started running. 
+    ![Run Log](Images/remote/run-log.png)
+    > **Note:** The job might be queued if there are insufficient resources available. If the log fails to display when the job starts, you can wait for a while and then execute the `AI Toolkit: Show the running fine-tuning job streaming logs` command to re-connect to the streaming log.
+    
+During this process, QLoRA will be used for fine-tuning, and will create LoRA adapters for the model to use during inference.
+The results of the fine-tuning will be stored in the Azure Files.
+
+## View and Query Logs on Azure
+
+After fine-tuning job was triggered, you can also view logs on Azure by clicking the "*Open Logs in Azure Portal*" button from the VSCode notification.
+
+Or, if you already opened the Azure Portal, you can find job history from the "*Execution history*" panel to the Azure Container Apps job.
+
+![Job Execution History](Images/remote/finetune-job-history.png)
+
+There are two types of logs, "*Console*" and "*System*".
+- Console logs, which are emitted by your app, including `stderr` and `stdout` messages. That's also what you may already see in previous section's streaming logs.
+- System logs, which are emitted by the Container Apps service, including the status of service level events.
+
+To view and query you logs, clicking the "*Console*" button and you will be navigated to the Log Analytics page. There you can view all the logs and write your own queries.
+
+![Job Log Analytics](Images/remote/finetune-job-log-query.png)
+
+
+> For more information about Azure Container App Logs, see [Application Logging in Azure Container Apps](https://learn.microsoft.com/azure/container-apps/logging).
+
+## Provisioning Components Included in the Template
+ 
+| Folder | Contents |
+| ------ |--------- |
+| `infra` | Contains all necessary configurations for remote operations. |
+| `infra/provision/finetuning.parameters.json` | Holds parameters for the bicep templates, used for provisioning Azure resources for fine-tuning. |
+| `infra/provision/finetuning.bicep` | Contains templates for provisioning Azure resources for fine-tuning. |
+| `infra/finetuning.config.json` |The configuration file, generated by the `AI Toolkit: Provision Azure Container Apps job for fine-tuning.` command. It is used as input for other remote command palettes. |
+
+## Advanced usage
+### Configuring Secrets for fine-tuning in Azure Container Apps
+Azure Container App Secrets offer a secure way to store and manage sensitive data within Azure Container Apps. This feature fulfills the need for a secure environment to handle sensitive data such as HuggingFace tokens and Weights & Biases API keys. If you need to set these values, AI Toolkit provides a command palette to input the secrets into the provisioned Azure container app job (as stored in `./finetuning.config.json`). These secrets are then set as **environment variables** in all the containers.
```

Comment:
Proposed:
Azure Container App Secrets provide a secure way to store and manage sensitive data within Azure Container Apps, like HuggingFace tokens and Weights & Biases API keys. Using AI toolkit's command palette, you can input the secrets into the provisioned Azure container app job (as stored in `./finetuning.config.json`). These secrets are then set as **environment variables** in all containers.

Reply:
Updated